projects/plugins/protect/changelog/fix-protect-in-progress-fixer-indicator

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Suggestion: I would use "patch" for this since it is a bug fix, will need to re-run `./tools/fixup-project-versions.sh` if you decide to change it.